### ISSUE TITLE
failing local tests

### ISSUE THREAD

**Issue by dankelley** [ArgoCanada/argoFloats#465]
I just pushed "develop" commit fe5a517c56f64c36897ed7604a90361688cfd8c4 but it is *not* ready for use, because it is failing local tests.  The odd thing is that if I run the tests in `tests/testthat/test_download.R` by clicking the "Run tests" button in RStudio, then no errors occur.

I'll look into this today.

```R
Error (test_download.R:20:15): getProfiles()
Error: 'subset' must be logical
Backtrace:
 1. testthat::expect_message(subset(index, 1:3), "Kept 3 cycles") test_download.R:20:14
 7. base::subset.default(index, 1:3)
──────────────────────────────────────────────────────
✓ |   3       | hex to bits                           
x |   5 2     | merge data [0.2 s]                    
──────────────────────────────────────────────────────
Error (test_merge.R:11:15): test merging core data
Error: argument "subset" is missing, with no default
Backtrace:
 1. testthat::expect_message(...) test_merge.R:11:14
 7. base::subset.default(...)
```

**Comment by dankelley** [comment]
Also, the 'merge' tests work fine when run individually.
```R
==> Testing R file using 'testthat'

ℹ Loading argoFloats
Loading required package: sf
Linking to GEOS 3.9.1, GDAL 3.3.0, PROJ 7.2.1
Loading required package: oce
Loading required package: gsw
Loading required package: testthat
Loading required package: s2

══ Testing test_merge.R ════════════════════════════════════════════════════════
[ FAIL 0 | WARN 0 | SKIP 0 | PASS 9 ] Done!

Test complete
```

**Comment by dankelley** [comment]
Fixed in "develop" commit  9e22a1b74f2db96d7370706af5df132fccd0d613, basically by undoing most of the crazy fiddling that I did over the weekend.  Local tests now work fine.
